readme.md

```diff
@@ -137,6 +137,7 @@ Customization Options:
 | cache_seconds | number    | manually set custom cache control           | 1800                 | 1800                | 1800                    |
 | count_private | boolean   | counts private contributions too if enabled | false                | N/A                 | N/A                     |
 | layout        | string    | choose a layout option                      | N/A                  | N/A                 | 'default'               |
+| card_width    | number    | set the card width                          | N/A                  | N/A                 | 300                     |
```

Comment:
Oooof i forgot to document card_width & i thought it was some git merge conflict issue so i removed it & now i reimplemented it. lol